src/tarantool_schema.c

```diff
@@ -226,6 +226,17 @@ strnindex(const char **haystack, const char *needle, uint32_t len, uint32_t hmax
 	return hmax;
 }
 
+/**
+ * Strict compare a null-terminated string with a length specified
+ * string.
+ */
+static inline bool
```

Comment:
You removed `inline` keywords in one of previous commits.

Reply:
Not all. Only unreasonable.

Reply:
Okay, I don't mind.